Core/Src/BMP.c

```diff
@@ -178,24 +178,21 @@ static bool _readCalibration(void)
         return false;
     }
 
```

Comment:
enums are by default ALL UPPER CASE
see: sBMP_state_t

Reply:
BMP is un unfortunate name as it is already a defined name for bitmap. This implies that this file has something to do with images.